examples/simple-process/README.md

```diff
@@ -0,0 +1,47 @@
+# Example: Simple Process
+
+A Simple Camunda 7 Process.\
```

Comment:
The `\` are not needed. GitHub and markdown in general keeps in a single paragraph all lines that are together. To create paragraphs you need to add an empty line between two text lines 👍🏾

Reply:
OK